### PR TITLE
St/staking docs

### PR DESCRIPTION
**Pull request by saratnt**
Jira ticket: https://horizenlabs.atlassian.net/browse/HZN-2083

### CODE REVIEW

File: docs/3-tutorials/01-run-collator.md [hunk 1]
```diff
@@ -1,5 +1,168 @@
...
+    - Open Polkadot.js Apps and connect it to the collator node.
+    - Select _Developer>RPC Calls_. 
+    - Select _author_ as endpoint and then _rotateKeys_. 
+    - Press _Submit RPC call_.
+    - Copy the string representing the public session key.
+- Map the collator account to the session keys created in the previous step:
+    - On Polkadot.js Apps, select _Developer>Extrinsics_.
+    - In _using the selected account_ field, select the collator account.
+    - Select _session_ and then _setKeys_.
+    - Put the session key saved before in _keys_ field and put _0x_ in _proof_ field.
```
Comment (i-skrypnyk) [inline docs/3-tutorials/01-run-collator.md:19]: I've checked the polkadot-sdk and it's indeed completely ignored. I feel fooled :D

File: docs/3-tutorials/01-run-collator.md [hunk 2]
```diff
@@ -1,5 +1,168 @@
...
+- In _using the selected account_ field, select the collator account.
+- Select _parachainStaking_ and then _scheduleCandidateBondLess_.
+- Set the _less_ field to the amount to be unbonded.
+- Submit the transaction. Follow the wizard and sign the transaction using the password you set for the account.
+
+### How to Execute a Scheduled Bond Less Request
+
+The steps for executing a bond-less request are:
+- After having scheduled a bond-less request, wait until _candidateBondLessDelay_ rounds have passed. The value for _candidateBondLessDelay_ can be found using Polkadot.js Apps, under _Developer>Chain State>Constants_ tab, selecting _parachainStaking_, _candidateBondLessDelay_ and pressing the "+" button.
+- In _using the selected account_ field, select the account that will request the extrinsic. It is not needed to be the collator account, it can be any account.
```
Comment (i-skrypnyk) [inline docs/3-tutorials/01-run-collator.md:101]: nitpick, but this is the only section where next steps are missing:
```
- On Polkadot.js Apps, select Developer>Extrinsics.
- In using the selected account field, select the collator account.
```

File: docs/3-tutorials/02-stake-and-delegation.md [hunk 3]
```diff
@@ -1,6 +1,163 @@
...
+
+Rewards for collators and their delegators are distributed after the reward payout delay and they are paid out on a block-by-block basis starting at the second block of the round. For every block, one collator will be chosen to receive their entire reward payout from the prior round, along with their delegators, until all of the rewards have been paid for that round.
+
+
+### General Tips on Selecting Collators
+
+- In general it is better to choose a collator with a lower total amount bonded, because in that case your delegation amount will represent a larger portion of the collator’s total stake and you will earn proportionally higher rewards. However, there is a higher risk of the collator being kicked out of the active set and not earning rewards at all.
+- Spreading delegations between multiple collators is more efficient in terms of rewards, but only recommended if you have enough to stay above the minimum stake of each collator.
+- You can consider collator performance by reviewing the number of blocks each collator has produced recently.
```
[the remark this answers is not in the document]
Comment (saratnt) [reply docs/3-tutorials/02-stake-and-delegation.md:28]: I don't think it is an info available with Polkadot.js, I suppose you need to use an explorer

File: docs/3-tutorials/01-run-collator.md (hunk 2 above)
Comment (saratnt) [reply docs/3-tutorials/01-run-collator.md:101]: Good catch, thanks!

File: docs/3-tutorials/02-stake-and-delegation.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (i-skrypnyk) [reply docs/3-tutorials/02-stake-and-delegation.md:28]: I was thinking about the Network -> Explorer tab in polkadot js, not the ideal thing but one can see what are the recent authors

**Review by paolocappelletti — APPROVED** [review]

**Pull request merged**